travel_platform/travel_app/views.py

```diff
@@ -3,10 +3,28 @@
 
 from django.http import Http404
 
+from django.http import JsonResponse
+
 from django.shortcuts import render
 
 from .models import Query
 
 
 def index(request):
     return render(request, 'travel_app/main.html')
+
+
+def coord(request):
+    """
+    View added for checking of interaction of map from landing page with 
+    server.
+    :param request: 
+    :return: 
+    """
+    if 'lon' in request.GET:
```

Comment:
creating fully working view is a different task than "integrate a map plugin into the project".
Added checking of 'lat' parameter.
What benefits "request.GET.get('lon')" gives us?
Generally I suppose turn work with this view into different task.